rfc/rfc-91/rfc-91-2.md

```diff
@@ -0,0 +1,75 @@
+
+# RFC-91-2: Storage LP conditional write retry handling
+
+Conditional writes with s3 which are retried due to transient issues are not idempotent, meaning that a successful PUT can go through but a retry can turn the response into a 412.
```

Comment:
It would be good to clarify the particular scenario and sequence of (network) events that lead to unexpected 412 on a conditional PUT that successfully writes an object on S3.  Then we can decide how to work around the behavior.